### DISCUSSION TITLE
Any way to make the selected tab more prominent?

### DISCUSSION THREAD

**Discussion by clmc99** [black7375/Firefox-UI-Fix#902] [Q&A]
In dark mode with Lepton Proton style, it's difficult to tell which tab is currently selected. The best I've found is setting ```userChrome.tab.photon_like_contextline``` to ```true```, but there's still not a lot of contrast (see below):
![image](https://github.com/black7375/Firefox-UI-Fix/assets/85983330/4054a3e0-525d-4984-9a38-3390a2c0acf5)
Is there a way to get more contrast between selected and unselected tabs, like in Edge (see below)?
![image](https://github.com/black7375/Firefox-UI-Fix/assets/85983330/7119c1c8-0699-4bd0-9514-4a60197d7184)

**Comment by clmc99** [comment]
I managed to get the current tab to appear gray by turning dark mode off and on again, but now the close tab button has disappeared

**Comment by black7375** [comment]
It's looks like this bug.
https://github.com/black7375/Firefox-UI-Fix/issues/865

1. Update to recent version!!

**Reply by clmc99** [reply]
I'm on the latest github version currently

**Reply by black7375** [reply]
Did you add or modify any settings in `about:configs`?
Also, what OS and theme are you using?

When I say theme, we're referring to this screen.
![image](https://github.com/black7375/Firefox-UI-Fix/assets/25581533/ea049059-f2d8-4be8-a3de-9b7f62b3fe25)

**Reply by clmc99** [reply]
I'm using the dark theme. I didn't modify anything in about:config, but I am using Betterfox. I went throught the Betterfox config and the vast majority of it has no overlap with Lepton, but I set up user.js so that any overlaps are overridden by Lepton. I was having this issue before I set up Betterfox, so I don't think that's the problem anyway
